python/ray/train/backends/torch.py

```diff
@@ -144,3 +145,144 @@ def on_shutdown(self, worker_group: WorkerGroup,
 
         worker_group.execute(
             shutdown_torch, destroy_process_group=len(worker_group) > 1)
+
+
+if torch:
```

Comment:
Hmm is there a cleaner way to do this without wrapping all the functions?

Reply:
One option is to just assume that torch is installed in torch.py, but move `TorchConfig` to backend.py

Reply:
Actually, maybe we don't even need backwards compatibility..after all this is still alpha. Just move all the files in `backends` package and just have a dummy `backends.py` that raises an error saying all the stuff has been moved.

Reply:
Did the refactoring in a separate PR: https://github.com/ray-project/ray/pull/20312